plugin/storage/es/mappings/jaeger-dependencies.json

```diff
@@ -1,8 +1,8 @@
 {
   "template": "*jaeger-dependencies-*",
   "settings":{
-    "index.number_of_shards": {{ .NumberOfShards }},
```

Comment:
Why do we need to change the mappings in this PR?  Isn't it simpler to change the field names in the class?

Reply:
having said that, ESIndex->IndexPrefix is a welcome change

Reply:
Yep,  I thought of doing the same but, as I was changing the files anyway for indexPrefix, so renamed other variables as well. Do you want me to revert it?

Reply:
Nah, let's keep it. If we have integration tests that exercise all these mappings, then it's a safe change